### ISSUE TITLE
Brazilian Portuguese Translation

### ISSUE THREAD

**Issue by LelePG** [summitech/gitexplorer#103]
Would it be possible to add a Brazilian Portuguese translation to the project? I've been using Git Explorer for a while now, and I'd love to contribute with a translation to Brazilian Portuguese, since it would be helpful to a lot of people I know, and would increase the amount of people who could use the project.

**Comment by bpires** [comment]
Hey @LelePG, this indeed would be really helpful to the Brazilian community. It's especially hard for beginners to memorize all the commands and the docs are not intuitive/simple enough for day-to-day usage.

The original project seems to be dormant. I think this is an awesome resource, and it's a bummer that they are not keeping it alive, but probably they got caught up in other projects and duties. The last commit is from 2 years ago and they aren't responding to pings. 

We could come up with an inspired version in Portuguese. Wanna team up? I reckon it's been a while since you put up this issue, but get in touch if you want to mature that idea. 

Either way, I'll try to contact the main collaborators (@ade-fatimah and @remiilekun) to talk about this issue or at least ask for permission to do a derivate project.

Cheers!

PS: I'm also on the Patocornio discord server, although not been very active there! 😄

**Comment by ade-fatimah** [comment]
Hi @bpires, apologies for the inactivity. How can we kickstart this?

**Comment by bpires** [comment]
Hi @ade-fatimah, thank you very much for responding! No need to apologize, I know how hard it can be to keep track of everything. 

First, congrats! You guys came up with an amazing and very useful project (and beautifully designed), please rely on the help of the community to keep it going. If you need assistance with maintaining it, post an issue about it and I'm sure there will be many people willing to collaborate.

Your quick response caught me by surprise 😅 but now that I know the project is indeed active, I will explore the repo in detail and mature the idea (besides the translation, we would need to include a language selector). 

When I have something concrete, I'll get back to you! Sounds good?

**Comment by ade-fatimah** [comment]
Yeah sounds good. Thanks a lot for your kind words and interest in the project. Looking forward to your contributions.

**Comment by LelePG** [comment]
Hi @bpires , unfortunately I'm kinda a short on time nowadays and will not be able to help. I wish you good luck with the project though. Once I have a reasonable amout of time to dedicate to the translation I'll check if that's still going and will try to help. 😄

**Comment by bpires** [comment]
No worries @LelePG! Thanks for responding though 😊

**Comment by Gzh0821** [comment]
Hey guys.😊
I am a beginner in React and Git. I searched for this project because of my interest. It seems that it has not been updated for a long time.
I found that it only has English pages, so I tried to fork the project and added the i18n localization function. If you are interested, we can add translations in various languages to the forked project (currently I have only added Simplified Chinese) translation content.
You can check out the examples in my forked project: [g_zh_gitexplorer](https://github.com/Gzh0821/g_zh_gitexplorer)
